packages/backend-core/src/sql/sqlTable.ts

```diff
@@ -210,16 +210,27 @@ function buildDeleteTable(knex: SchemaBuilder, table: Table): SchemaBuilder {
 
 class SqlTableQueryBuilder {
   private readonly sqlClient: SqlClient
+  private extendedSqlClient: SqlClient | undefined
```

Comment:
Ooh how come this is necessary? Why can't we just use `sqlClient`?

Reply:
We need to know that the implementation/dialect is `MY_SQL` - but also its MariaDB - for Knex it needs to know its MySQL, technically we could get around this by setting everywhere that needs to know its MySQL to MariaDB when needed, but I like the idea of separating the difference in client here, since its still a MySQL datasource, it just reads a different version once it connects.